imported/spirv/spvgen.h

```diff
@@ -240,11 +240,9 @@ void SH_IMPORT_EXPORT vfxPrintDoc(
 }
 #endif
 
-static inline bool InitSpvGen(
-    const char* pSpvGenDir = nullptr)
-{
-    return true;
-}
+bool InitSpvGen(const char* pSpvGenDir = nullptr);
```

Comment:
This is not related to this PR per se, but why is the spvgen header duplicated in LLPC?

Reply:
we didn't provide the spvgen directory before, so import this file.

Reply:
I don't follow. The spvgen file has been in spvgen for a very long time https://github.com/GPUOpen-Drivers/spvgen/commits/dev/include. Why do we need a second copy within llpc?

Reply:
Yes, this file is unnecessary. This file is historical issue, we have plans to remove it.